content/agent/_index.md

```diff
@@ -3,103 +3,137 @@ title: Agent
 
-#### Check your machine's time
-We have also seen a few cases where machines have their clock set further in the future or the past, which can sometimes cause problems with metric submission.
-To check for this, run:
-
-```shell
-date -u && curl -s -v https://app.datadoghq.com 2>&1 | grep Date
-```
-This outputs the current system’s date, and then makes a request to our endpoint and grabs the date on our end.
-If these are more than a few minutes apart, you should correct the time settings on your server.
+4. Wait a few minutes to generate some logs. [Look here](/agent/#log-locations) to find the location of the logs.
 
-### Issues getting integrations working
+### Send a flare
 
-Datadog has many [integrations](/integrations/) which are set up through [YAML files in the Agent](https://github.com/DataDog/dd-agent/tree/master/conf.d).
 
-Here is a quick guide for troubleshooting [integrations](/integrations) installation:
+If you are running the 5.3 version (or higher) of the agent, you're able to send all necessary troubleshooting information to our Support Team, with one flare command!
 
-1. [Run the info command](/agent/faq/agent-status-and-information).
+`flare` gathers all of the agent's configuration files and logs into an archive file. It removes sensitive information including passwords, API keys, Proxy credentials, and SNMP community strings.  
+**Confirm the upload of the archive to immediately send it to Datadog support**.  
+Since the Datadog Agent is completely open source, you can [verify the code's behavior](https://github.com/DataDog/dd-agent/blob/master/utils/flare.py). You can also review the archive prior to sending as the flare prompts a confirmation before uploading it.  
 
-2. Is the integration showing up in the [info command](/agent/faq/agent-status-and-information)?
+In the commands below, replace `<CASE_ID>` with your Datadog support case ID, if you don't specify a case ID, the command asks for an email address that is used to login in your organization and creates a new support case.
 
-* **No, it's not**:
-    * Check the configuration file, make sure it is in the right location and named correctly.
-    * Check it in a YAML parser to make sure it has the correct syntax. [Example files](https://github.com/DataDog/dd-agent/tree/master/conf.d).
-    * If you moved or changed the file, [restart the Agent](/agent/faq/start-stop-restart-the-datadog-agent) and rerun the [info command](/agent/faq/agent-status-and-information) to see if it is now showing up.
+|Platform|Agent v5 |Agent v6|
+|:--------|:-----|:--------|
+|Linux| `sudo /etc/init.d/datadog-agent flare <CASE_ID>` | `sudo -u dd-agent -- datadog-agent flare <CASE_ID>`|
+|Docker|`docker exec -it dd-agent /etc/init.d/datadog-agent flare <CASE_ID>`||
+|Docker (Alpine)|`docker exec -it dd-agent /opt/datadog-agent/bin/agent flare <CASE_ID>`||
```

Comment:
same considerations about the meaning of an empty cell

Reply:
👌